src/modules/scenes/FirstScene.tsx

```diff
@@ -1,64 +1,125 @@
-import { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button, Stack, Typography } from '@mui/material';
 
 import { OrbitControls } from '@react-three/drei';
 import { Canvas } from '@react-three/fiber';
 
+import { SIMULATION_CSV_FILE, SIMULATION_FRAME_MS } from '@/config/simulation';
 import { SeasonProvider, useSeason } from '@/context/SeasonContext';
+import { SimulationProvider, useSimulation } from '@/context/SimulationContext';
+import { TRANSLATIONS } from '@/langs/constants';
+import { SimulationStatus } from '@/types/simulation';
+import { formatHoursToDays } from '@/utils/time';
 
 import { Forest } from '../models/Forest';
 import { Garden } from '../models/Garden';
 import { House } from '../models/House/House';
 import { Tree } from '../models/Tree/Tree';
 
 const FirstSceneComponent = (): JSX.Element => {
-  const { nextSeason } = useSeason();
+  const { t } = useTranslation();
+  const { season } = useSeason();
 
-  // TODO: Demo only, will be removed
-  useEffect(() => {
-    const intervalId = setInterval(() => {
-      nextSeason();
-    }, 3_000);
+  const {
+    indoorTemperature,
+    outdoorTemperature,
+    period,
+    progression,
+    heatLoss,
+    startSimulation,
+    status,
+    totalHeatLoss,
+    electricityCost,
+    duration,
+  } = useSimulation();
 
-    return () => {
-      clearInterval(intervalId);
-    };
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  const formattedHours = formatHoursToDays(period.durationInHours);
 
   return (
-    <Canvas
-      style={{ height: '100vh', width: '100vw' }}
-      camera={{ position: [10, 1, -15], fov: 75 }}
-    >
-      <color attach="background" args={[0x87ceeb]} />
-      {/* Ambient Light for overall illumination */}
-      <ambientLight intensity={1.2} />
-      {/* Main Sunlight Simulation */}
-      <directionalLight
-        position={[6, 30, -10]}
-        intensity={2}
-        color={0xffffff}
-      />
-      <OrbitControls
-        enableDamping
-        dampingFactor={0.05}
-        minPolarAngle={Math.PI / 6} // Minimum angle
-        maxPolarAngle={Math.PI / 2.5} // Maximum angle
-        enableZoom={false}
-        maxDistance={150}
-      />
-      <House position={[0, 0, 0]} />
-      <Garden position={[0, -0.5, 0]} />
-      <Tree position={[6, 0, -2]} />
-      <Forest position={[6, 0, 1]} />
-    </Canvas>
+    <>
+      <Stack position="absolute" zIndex={1} spacing={1} p={2}>
+        <Button
+          variant="contained"
+          onClick={startSimulation}
+          disabled={status === SimulationStatus.RUNNING}
+        >
+          {status === SimulationStatus.RUNNING ? 'Running...' : 'Start'}
+        </Button>
+        <Stack spacing={1}>
+          <Stack>
+            <Typography variant="h6">
+              Simulation duration:{' '}
```

Comment:
Translate?

Reply:
I will remove this because it was just for debugging.